publish.gradle

```
@@ -0,0 +1,11 @@
+apply plugin: "maven-publish"
```

Use a short name

---

This name is for the Gradle built-in plugin: https://docs.gradle.org/current/userguide/publishing_maven.html

That plugin is applied by https://github.com/vanniktech/gradle-maven-publish-plugin which is the plugin we want to use. The old code keyed off of `maven-publish` because it needed to wait until the `publishing` block was available. Since the application of the plugin is now in the same file, we don't need to wait.
```suggestion
apply plugin: "com.vanniktech.maven.publish"
```